### PR TITLE
Generate Ubuntu tests and remove the old manually created ones.

### PR DESCRIPTION
**Pull request by yguo0905**
Generate Ubuntu tests and remove the old manually created ones.

/assign @krzyzacy

### CODE REVIEW

File: experiment/generate_tests.py [hunk 1]
```diff
@@ -53,7 +53,7 @@
           value: /etc/ssh-key-secret/ssh-private
         - name: JENKINS_GCE_SSH_PUBLIC_KEY_FILE
           value: /etc/ssh-key-secret/ssh-public
-        image: gcr.io/k8s-testimages/kubekins-e2e-prow:v20170720-69a2043d
+        image: gcr.io/k8s-testimages/kubekins-e2e-prow:v20170725-1044fad2
```
Comment (krzyzacy) [inline experiment/generate_tests.py]: it will be https://github.com/kubernetes/test-infra/pull/3717 shortly

Comment (yguo0905) [inline experiment/generate_tests.py]: Just curious. There's a reason not to use `latest`?

Comment (krzyzacy) [inline experiment/generate_tests.py]: canary job uses latest, we wait till they pass then push to prod. Kind of staging process here.

Comment (yguo0905) [inline experiment/generate_tests.py]: Sounds like we could have another `stable` tag.

Comment (krzyzacy) [inline experiment/generate_tests.py]: yeah we are discussing it..

Comment (krzyzacy) [inline experiment/generate_tests.py]: I'll open an issue

**Comment by yguo0905** [comment]
Rebased. PTAL!

**Comment by krzyzacy** [comment]
/lgtm

ah now you need to rebase it again

**Comment by yguo0905** [comment]
Rebased. Should be good to go.

**Comment by yguo0905** [comment]
Rebased. PTAL.

**Comment by krzyzacy** [comment]
whoops sorry forgot to push the button

**Pull request merged**